packages/wonder-blocks-timing/src/util/__tests__/action-scheduler.test.js

```diff
@@ -66,6 +66,7 @@ describe("ActionScheduler", () => {
                 actionScheduler.clearAll();
 
                 // Assert
+                // $FlowIgnore[method-unbinding]
                 expect(testTimeout.clear).toHaveBeenCalledWith(true);
```

Comment:
And here?  Does Flow just not like `.toHaveBeenCalledWith()` and `.toHaveBeenCalledTimes()` calls?

Reply:
It has nothing to do with the calls. It sees any method that is referenced without being called as a potential unbinding right now.

Reply:
That makes sense.